config.yml.dist

```diff
@@ -0,0 +1,12 @@
+oauth2:
+  clients:
+    client-name:
+      id: id
+      secret: secret
+      redirect_uri: http://www.example.com/
```

Comment:
Normally multiple redirect locations are allowed, don't think that we've ever need more than 1 though?